src/com/moclojer/webhook.clj

```diff
@@ -5,19 +5,23 @@
 
 (defn request-after-delay
   "after a delay call http-request, return body"
-  [{:keys [url method body headers sleep-time]
+  [{:keys [url method body headers sleep-time condition]
     :or {headers {}
+         condition true
```

Comment:
since the webhook condition will return true if it is nil, does it make sense to add the default condition true here? this is just a question because we are adding here, bo make explicit and the function webhook-condition could do this and make it explicit in one place?  if I understood the idea